package.json

```diff
@@ -79,7 +79,7 @@
     "babel-preset-env": "^1.3.3",
     "chai": "^4.0.2",
     "coveralls": "^3.0.0",
-    "eslint": "latest",
```

Comment:
Having latest should be fine. This is only for dev dependencies.

Reply:
Great! Sure 👍